README.md

```diff
@@ -140,3 +140,48 @@ with
 
 
     bundle update github-pages
+
+
+#### Using docker container for local builds ####
+
+**Problems with installing jekyll/github-pages?** Try a docker
```

Comment:
I would frame this as an alternative approach - actually maybe it should be the primary one?

Reply:
ok, I'll rewrite

Reply:
See 5d95a0f